### ISSUE TITLE
[Suggestion] In regards to possible future scaling.

### ISSUE THREAD

**Issue by SnowyBirdot** [tsunamayo/Starship-EVO#3724]
Hello Tsuna, I hope you're doing well, given what must be a fairly large workload as of current.

I bring with me, a suggestion that is by, and large totally unnecessary, and possibly a bit silly depending on your opinion. That being, the ability to scale blocks, smaller than a brick!

![20210407155153_1](https://user-images.githubusercontent.com/72358669/113940158-3ee77700-97ba-11eb-85e5-89910be4892c.jpg)

As illustrated in this handy screenshot, which shows a block, a brick, and an optical illusion of a block roughly 1/2 the size of a brick.

Now you may wonder in what world this would be vaguely needed. It is not.
It is however something I've decided to suggest for little reason other than my addiction to increasingly small, complex mechanisms, and the upcoming building rework, which has led to my wondering.

Could we scale smaller than .25?

So I've decided to suggest this! And I'm not the only interested party, though there probably aren't many others, especially on Github.
You may know me from a few bug reports, and that one attached blueprint of a fighter with 50 child entities. Which likely gives you a solid idea of why I would be interested in something such as this. 

So far as I'm aware, we already can build weapons with blocks smaller than bricks, so could I get your thoughts on such a thing for normal building? I'm not really expecting a yes, but I am interested in hearing what you think about it. After all, you are the developer and do quite a good job of it. 

(This has been peer-reviewed, the majority response being, "Dew it".)

**Comment by ultrak2k** [comment]
I would absolutely love microblocks for superfine detailing and weapons.

No I don't think spam of them are a problem because who would seriously have the endurance to build anything that finely detailed en masse, even to copy and paste

**Comment by TIKIRobo** [comment]
Dew it

but in all seriousness this would be really helpful for handheld weapons as well as drones which normally tend to be pretty bland

**Comment by SnowyBirdot** [comment]
Yay, support from people who like the idea has been garnered.

**Comment by glorbo703** [comment]
We have to remember that this is a game and games have limitations, by doing this you're just inviting yourself to want even smaller bricks to build with for even more detail. If you want to remove limitations from the building process it seems like what you actually want isn't sevo, but rather a 3D editing software.

Tldr: just use blender lol

**Comment by SnowyBirdot** [comment]
We have all been memeing it up in the discord for some time. I mostly just want Tsuna's thoughts. It's about 90% meme, 10%, it'd be neat.

**Comment by RockefellerDoctor** [comment]
> 
> 
> We have to remember that this is a game and games have limitations, by doing this you're just inviting yourself to want even smaller bricks to build with for even more detail. If you want to remove limitations from the building process it seems like what you actually want isn't sevo, but rather a 3D editing software.
> 
> Tldr: just use blender lol

tbh i wouldn't mind something like a 3D editor in SEVO, at least for guns. The current block system for guns is clunky, chunky, and not very nice.

**Comment by ultrak2k** [comment]
> We have to remember that this is a game and games have limitations, by doing this you're just inviting yourself to want even smaller bricks to build with for even more detail. If you want to remove limitations from the building process it seems like what you actually want isn't sevo, but rather a 3D editing software.
> 
> Tldr: just use blender lol

One does have to note that we can’t really ask for any brick smaller because the charachter has a determined size and it’s not like we can shrink down 
Also while understandable, that line of reasoning is a slippery slope fallacy I *think* but yeah there does come a point where just use blender, but it’s not like you can really use blender to do cool shit like in a game unless you’re minusT or someone else crazy

**Comment by tsunamayo** [comment]
Yes I have though about this also, I will first see how things go. Again it might be shooting myself in the foot with the upcoming prefab tool... It could come in handy for guns indeed.
@glorbo703 I have said myself this a thousand time but refrained myself from saying it. At one point we have to say stop it is not an editing software...
Cheers

**Comment by cloudrambler** [comment]
@tsunamayo I agree with the principle that as a GAME, there should be a few awkward quirks to work around, otherwise there is less challenge and less payoff for putting in the effort. However I will give you one clear example of why smaller blocks would be valuable. I spend a lot of time refining the shape of my ships to make them as clean as I can manage, often to suit a design that I have roughed out as set of sketches or models. Sometimes I have to make ships larger than I really want them to be in order to fit in the minimum number of bricks to convey the shape kindly. If the blocks were smaller, then I could make the same ship smaller, and it would feel more human scale inside.

**Comment by SnowyBirdot** [comment]
> Yes I have though about this also, I will first see how things go. Again it might be shooting myself in the foot with the upcoming prefab tool... It could come in handy for guns indeed.
> @glorbo703 I have said myself this a thousand time but refrained myself from saying it. At one point we have to say stop it is not an editing software...
> Cheers

I appreciate getting to hear your thoughts on it Tsuna. Hope life is treating you well.

**Comment by tsunamayo** [comment]
@SnowyBirdot Hi, so now we have mini bricks. Thanks
